src/services/purchaseService.ts

```
@@ -154,13 +154,14 @@ const handleSlotPurchase = async (
     typesName: string,
     accountId: string,
-    quantity: number
+    quantity: number,
+    isStorePurchase: boolean = false
 ): Promise<{ InventoryChanges: IInventoryChanges }> => {
     const typeCategory = getStoreItemTypesCategory(typesName);
     logger.debug(`type category ${typeCategory}`);
     switch (typeCategory) {
         default:
-            return await addItem(accountId, typesName, quantity);
+            return await addItem(accountId, typesName, quantity, isStorePurchase);
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)